### DISCUSSION TITLE
Poseidon - menu - expanded="false"

### DISCUSSION THREAD

**Discussion by upsomar** [primefaces/community#81] [PrimeFaces Templates]
How can I make the menu smaller?
By default it is displaced and I cannot pick it up.

**Comment by cagataycivici** [comment]
Can you post a screenshot please? It has a couple of menu modes like horizontal, overlay ...

**Comment by mertsincan** [comment]
Closed for now! If an update comes from you, please reopen this ticket. Thanks a lot for your understanding!

Best Regards,
